specs/Spec.Hash.Definitions.fst

```diff
@@ -55,6 +77,8 @@ let max_input_length: hash_alg -> Tot pos = function
   | MD5 | SHA1
   | SHA2_224 | SHA2_256 -> pow2 61 - 1
   | SHA2_384 | SHA2_512 -> pow2 125 - 1
+  | Blake2S -> pow2 64 - 1
+  | Blake2B -> pow2 128 - 1
```

Comment:
Doesn't conflict anymore now (see above)... Btw, I wondered why it didn't make the proofs fail, because there indeed is a conflict between `blake2b` and `max_length`. What happens is that in the spec proofs we always use the generic `blake2` which uses `max_limb` (which is correct), and in the implementation we either use buffers whose lengths are smaller than 2^32 (when we hash all the data at once), or we incrementally process data whose length is given by the RFC, but once again we use the generic blake2 spec...